assets/airports/ktus.json

```diff
@@ -170,9 +170,9 @@
                 "BAYBI": ["BAYBI"],
                 "BBALL": ["BBALL"],
                 "GBN": ["ROSKR", "GBN"],
-                "MESCA": ["MESCA", "SSO"],
+                "MESCA": ["MESCA"],
                 "PXR": ["BBALL", "PXR"],
-                "REDDY": ["REDDY", "SSO"]
+                "REDDY": ["REDDY"]
```

Comment:
I don't mean to go back and forth on you, but now that I think about it again, we _should_ be able to deal with this in all likelihood. It looks up the _name_ of the exit, and then generates the waypoints. Whether or not the last waypoint generated _actually ends up being_ the name of the exit shouldn't matter. :thinking:

Do you remember if everything seemed to work when it was still implemented as charted (`"MESCA": ["MESCA", "SSO"],`, etc?

Reply:
Correct, we just need the name of the exit. Whether or not it's last in the list dies not matter.

Reply:
Done 👍